plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -439,11 +439,44 @@ public void testDecimal()
                 .execute(getQueryRunner(), clickhouseCreateAndInsert("tpch.test_nullable_decimal"));
     }
 
+    private SqlDataTypeTest addBinaryRoundTrip(SqlDataTypeTest sdtt, String inputType)
```

Comment:
No need to take SqlDataTypeTest as the argument. See `TestPostgreSqlTypeMapping#arrayDecimalTest` as the example. 

Also, please avoid abbreviations. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-abbreviations

Reply:
Addressed, thanks.